projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.ts

```diff
@@ -265,4 +265,8 @@ export class NavigationUIComponent implements OnInit, OnDestroy {
 
     this.isOpen = this.openNodes.length > 0;
   }
+
+  getIcon(title: string): string {
+    return ICONS[title] || '';
+  }
```

Comment:
[MEDIUM]
Can titles of navigation nodes be translated to various langauges? If yes, then the `ICONS` binding that we have created in another file, won't work.
Instead, I believe the backend should provide an information about an icon (an abstract icon code/name?), which could be potentially bind and displayed in the UI.

Reply:
currently we don have any attribute for Navigation Node in the backend which could support information for Icon. Please suggest.

Reply:
As far as I remember, the left-column menu in the Enanced MyAccount is a new CMS-driven component. Can we provide there any information about the icon to be used? Or any unique semantic name for each link? ... so we could bind icons to it, instead of being based on link title (which I guess can have different versions in various langauges)

Reply:
yes, we can make enhancements in the backend.. but the FP3 tasks (Commerce stories)are already closed from our side . Any enhancement to the implementation can be taken only in the next release(december)....but we want My-account to be delivered in this spartacus release only

Reply:
I have changed the function now to get Icon based on the navigation node id to get rid of various languauges